src/Akka.Persistence.Sql/Config/EventJournalTableConfig.cs

```diff
@@ -10,13 +10,16 @@ namespace Akka.Persistence.Sql.Config
 {
     public sealed class EventJournalTableConfig : IEquatable<EventJournalTableConfig>
     {
-        public EventJournalTableConfig(Configuration.Config config)
+        public EventJournalTableConfig(Configuration.Config config, string tableMapName)
```

Comment:
We need to know if we're using the default table mapping or if we're running in a backward compat mode. WriterUuid is only available in Akka.Persistence.Sql and not in any other legacy plugins.